README.md

```diff
@@ -102,103 +73,39 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Creating tables in SQL**
+- **Inserting tables in SQL**
+- **Querying tables in SQL**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🚀 Screenshots of Project <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+> screenshots of the project
 
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+![Screenshot](./1.jpg)
+![Screenshot](./2.jpg)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
-
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
 ### Setup
 
 Clone this repository to your desired folder:
 
```

Comment:
- [ ] Kindly add step by step how to make use of your app on the user local machine. Your README file lacks this steps at this moment.